doc/dev/crimson/crimson.rst

```diff
@@ -144,86 +122,36 @@ one of the two following configuration options:
 Running Crimson
 ===============
 
-As you might expect, Crimson does not yet have as extensive a feature set as does ``ceph-osd``.
```

Comment:
The below diff is very ugly to follow, I recommend checking out the rich diff instead